examples/Demo/FluentUI.Demo.Client/Documentation/Components/TextArea/FluentTextArea.md

```diff
@@ -0,0 +1,50 @@
+---
+title: TextArea
+route: /TextArea
+---
+
+# TextArea
+
+A **FluentTextArea** component enables a user to enter multiple lines of text.
+The component is a wrapper around the FluentUI `TextArea` component.
+
+## Appearance
+
+The apparent style of a textarea can be changed by setting the `Appearance` property, but also by setting the `Size` property.
```

Comment:
The apparent style of a textarea can be changed by setting the `Appearance`  and  `Size` properties.